src/odemis/driver/test/smartpod_test.py

```diff
@@ -42,18 +42,17 @@
         "role": "",
         "locator": "usb:ix:0",
         # "locator": "fake",
-        "options":"",
         "axes": {
             'x': {
-                'range': [-1, 1],
+                'range': [-0.2, 0.2],
                 'unit': 'm',
             },
             'y': {
-                'range': [-1, 1],
+                'range': [-0.2, 0.2],
                 'unit': 'm',
             },
             'z': {
-                'range': [-1, 1],
+                'range': [-0.1, 0.1],
                 'unit': 'm',
             },
             'theta_x': {
```

Comment:
Do you have better range of the rotation axes too? In particular, can't they go to negative value? I'd tend to think that 0 is the default position, and it can go in both direction? The range also seems a lot less than 180°.